mmdet/models/losses/iou_loss.py

```diff
@@ -49,18 +55,24 @@ def iou_loss(pred, target, linear=False, mode='log', eps=1e-6):
 
 
 @weighted_loss
-def bounded_iou_loss(pred, target, beta=0.2, eps=1e-3):
+def bounded_iou_loss(pred: Tensor,
+                     target: Tensor,
+                     beta: float = 0.2,
+                     eps: float = 1e-3) -> Tensor:
     """BIoULoss.
 
     This is an implementation of paper
     `Improving Object Localization with Fitness NMS and Bounded IoU Loss.
     <https://arxiv.org/abs/1711.00164>`_.
 
     Args:
-        pred (torch.Tensor): Predicted bboxes.
-        target (torch.Tensor): Target bboxes.
-        beta (float): beta parameter in smoothl1.
-        eps (float): eps to avoid NaN.
+        pred (Tensor): Predicted bboxes.
```

Comment:
suggest to also indicate the shape of the tensor

Reply:
Yes, i have specified the shape of inputs.